build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -117,19 +121,33 @@ Future _createDevCompilerModule(
     request.arguments.add('--no-source-map');
   }
 
-  // Add all the linked summaries as summary inputs.
+  // Add linked summaries as summary inputs.
+  //
+  // Also request the digests for each and add those to the inputs.
+  //
+  // This runs synchronously and the digest futures will be awaited later on.
```

Comment:
Once we get rid of the `useKernel` branches we should be able to clean this up with the same pattern as above...

Reply:
yep, will do